inst/eDISH_app/server.R

```diff
@@ -199,6 +219,29 @@ function(input, output, session){
   })
 
   
+  # passing parameters for knitting on export button click. Call when chart generated
+  observeEvent(status2()==TRUE,
```

Comment:
We should trigger this from the click of the download button.  But I agree that status(2)==TRUE should be a requirement to receive the chart/report.